mods/rv/rules/soviet-structures.yaml

```diff
@@ -1828,6 +1828,10 @@ naexpn:
 	Inherits: ^ExpansionNode
 	# Transforms:
 	#	IntoActor: exps
+	Inherits@AI: ^AIMicroManagable
```

Comment:
This should go above the commented out transforms.

Reply:
fixed